shell/server/proxy.js

```diff
@@ -880,6 +881,8 @@ function getApiSessionParams(request) {
 getProxyForApiToken = (token, request) => {
   check(token, String);
   const hashedToken = Crypto.createHash("sha256").update(token).digest("base64");
+  const tabId = Crypto.createHash("sha256").update("tab:").update(hashedToken)
```

Comment:
Maybe ApiSessions shouldn't all share the same tabId?

Consider the Piwik case - all of the requests from the tracked page will share the same `tabId`, even though they come from different browsers, different IPs, and will pass the requests on to different `Proxy`s (due to the `apiSessionProxies.get() call below`).